core/src/main/java/bio/terra/pearl/core/service/kit/PepperDSMClient.java

```diff
@@ -19,5 +18,5 @@ public interface PepperDSMClient {
      */
     String sendKitRequest(Enrollee enrollee, KitRequest kitRequest, PepperKitAddress address);
     PepperKitStatus fetchKitStatus(UUID kitRequestId);
-    Collection<PepperKitStatus> fetchKitStatusByStudy(UUID studyId);
+    Collection<PepperKitStatus> fetchKitStatusByStudy(String pepperStudyName);
```

Comment:
Pepper wants a study name, not the Juniper study UUID.

Reply:
how does this name get set?